scripts/domain/SellerFields.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Boson Protocol Domain Enum: SellerFields
+ */
+class SellerFields {}
```

Comment:
Perhaps this can be named `SellerUpdateFields`, since it is very specific to seller updates.